app/client/components/ForgetPassword.tsx

```diff
@@ -0,0 +1,110 @@
+'use client'
+import Image from 'next/image'
+import { XMarkIcon } from '@heroicons/react/24/solid'
+import Link from 'next/link'
+import { useState } from 'react'
+
+export default function ForgetPassword() {
+  const [email, setEmail] = useState('')
```

Comment:
Add generic type to useState like 
```suggestion
  const [email, setEmail] = useState<string>('')
```